packages/Checkbox/stories/tests/Screener.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
I think it would be better to make this the `Variations.js` story, include it in the Storybook at the root level for the Checkbox (for documentation consumers), and then re-use it as the "Backyard/Tests/Screener" story as well for test coverage.